### PR TITLE
medical rework

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Drops my medical rework as a draft while I tinker on it, so people can follow on and ~~Get mad~~ see what they want to know.

Overall this aims to rebalance medicine to be less chemical based and more tool based, and reduce the requirement of premedding, and in the future making it not required to play the game.

Adds a few things such as new splints, more pill packets.

Removes certain pill bottles from medvendors, they'll stay MD and Medic only items for these chemicals, these chemicals are- Tricord, Bicard, Kelotane and Tramadol.

Adds a packet section to the medvendor, infinte, some utility pills are packeted if you want it packeted.

Misc pill bottles that aren't removed are now infinite. No reason not to have them be infinite.

Bandages and Oinement heal 15 on use, but take a bit longer to use. And heal .75 from .1 overtime.

Rebalances chemicals overall, most chemicals work faster but do more toxin damage.

Bicardine/Kelo heal faster, but deal more toxin damage at higher units, and deal a tiny amount of toxins at base.

Tramadol unchanged, deals a bit of toxin damage.

Tricord nerfed, it has no downsides.

Neuraline purges the t1 trio, because I have no idea if I should keep it in the game so this will do for now.

Hyro heals less toxin overtime.

Medical Nanites are far less lethal, considering premed has been nerfed. 1/4th blood loss, halved damage overall.

If I've missed any I've changed, or I'm wrong. Feel free to tell me and I'll update this.

Pill packets homogenized to 4, no reason to have them higher than usual IMO. And the pills that did get homogenized generally will give you more units.

Certain pill packets are small sized to prevent pill bottle stacking, but have more units in them.

First aid kits get bandages and get a size up, this doesn't change much. As medkit pouches are also 7.

Emergency splint sprite is bad, and subject to change, but they don't require skills and I think they're interesting to check out.

General Medical supplies are made infinite.

Bandages/Ointment heal 15 damage, and treated limbs heal 1 per tick when they hit the prob roll.

Defibs heal more. 
0/4/8/12/16/20

new-0/6/12/18/24/30

## Why It's Good For The Game
Makes medical less chemical based and more toolbased, and makes taking medicine for absurds amounts of time dangerous, as you need to be on chems for absurd amounts of times to take much toxin damage. I feel like this diversifies

## Changelog
:cl:
add: Emergency Splints, they don't require skill to use, but there's far less of them in the stack, and they're easy to apply to yourself.
balance: First aid kits get bandages and get a single more storage slot.
balance: removed pill bottles from med vendor, bandages and ointment heal 15 damage on use, but take much longer to use, advanced heal packs heal 30 if you have the skills for it. Treated wounds heal much faster than they used to, healing one per tick.
balance: Medical has had a large chemical rebalance. The main changes are T1 Trio deal minor toxin damage, heal more at higher doses but deal toxin damage in turn. Neuraline purges all T1 Trio for all effects go to #9236 to find everything. Some other notes are that medical nanites deal less overall damage, which sould make them still useable post nerf.
balance: defibs heal more- 0/6/12/18/24/30 from 0/4/8/12/16/20
/:cl:

### CODE REVIEW

**Comment by novaepee** [comment]
If marines can't cope, skill issue. Nice PR.

File: code/game/objects/machinery/vending/marine_vending.dm
Comment (novaepee) [inline code/game/objects/machinery/vending/marine_vending.dm:786]: Bocket, found by hyper

File: code/game/objects/items/storage/pill_packets.dm
Comment (novaepee) [inline code/game/objects/items/storage/pill_packets.dm:41]: Not actually ryetalyn as commented by Poofy 2 User in Discord.

**Comment by private-tristan** [comment]
will this include making pills MD/corpsman only openable, like CM? or are marines able to loot them off of corpsmen's bodies/ get it handed out to them
will half the marine force still to use chemistry so that they can get some chems? 
8 seconds seems like a **lot** of time for 10 extra damaged healed on a limb, maybe drop it to 4 or 5?

File: code/game/objects/items/stacks/medical.dm
Comment (XSlayer300) [inline code/game/objects/items/stacks/medical.dm:289]:
```suggestion
	desc = "An emergency medical splint created for emergency situations, easy and quick to apply, but not very durable."
```

**Comment by novaepee** [comment]
> will this include making pills MD/corpsman only openable, like CM? or are marines able to loot them off of corpsmen's bodies/ get it handed out to them will half the marine force still to use chemistry so that they can get some chems? 8 seconds seems like a **lot** of time for 10 extra damaged healed on a limb, maybe drop it to 4 or 5?

In #dev-general, 
image

I'm assuming he wants to make it long so that you have to be really out of combat.

**Comment by AndroidMK2** [comment]
Make it so researchers get some pill bottles too

**Comment by novaepee** [comment]
A stack of gauze/ointment or advance brute/burn kit only has 10, and a marine has 11 targets to get healed.

Buff stack to at least 15 or more.

**Comment by Mrrpip** [comment]
Eyo, while niche and rare, it may be a good idea to update the Tajaran gauze/ointment variants as they can be obtained trough hydroponics (Via a hacked seed vendor).

(Though a more classic gauze/ointment could be fun with your idea, A.E instant but less compared to your gauze.)

**Comment by Losenis** [comment]
>Ctrl f "kit"
>No results in pr

Make them apply to the entire body per use, they're nigh-useless save for corpses over 200 dmg. So long as there's no real actual alternative to chems they'll always be in an awkward position where any change to them won't make marines use them any less.

**Comment by EvilMelonTheSecond** [comment]
Chems still heal nothing due to tox damage. This PR requires you to premed harder to counter the shitty tox side effects and will only end up with marines who don't powergame feeling the nerf while people who minmax chems still doing the same premedding but with dylo/hyro mixed in.

**Comment by Pariah919** [comment]
I can't think of anything else to change without testing, so I guess this needs to be tested.

**Comment by Losenis** [comment]
Funnily enough I can picture this pr making marines rely even harder on chems and premed

File: code/game/objects/items/stacks/medical.dm
Comment (BraveMole) [inline code/game/objects/items/stacks/medical.dm:179]: I think this has too much consequences on corps trying to defib someone. 

What about doing this BUT giving corp like a tool to heal dead bodies?

File: code/game/objects/items/storage/pouch.dm
Comment (BraveMole) [inline code/game/objects/items/storage/pouch.dm:125]: packet only now then

File: code/game/objects/machinery/vending/marine_vending.dm
Comment (BraveMole) [inline code/game/objects/machinery/vending/marine_vending.dm:792]: i disagree with having all of those as pill bottles, i think the choice you gave as pill packet was fine.

I think it's missing innapro and dexalin in pill packet, but remove those bottles

File: code/game/objects/items/storage/pill_packets.dm
Comment (novaepee) [inline code/game/objects/items/storage/pill_packets.dm:43]: dylovene pill packet's desc says that it contains ryetalyn, which it does not

File: code/game/objects/items/stacks/medical.dm
Comment (BraveMole) [reply code/game/objects/items/stacks/medical.dm:179]: Or making a device in medbay to heal bodies back to 200 maybe (a cryo change?)

File: code/modules/organs/limbs.dm
Comment (BraveMole) [inline code/modules/organs/limbs.dm:596]: what give this treating status?

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/items/reagent_containers/pill.dm
Comment (novaepee) [inline code/game/objects/items/reagent_containers/pill.dm:249]: I don't understand why these have 20u when it was established that 10u give toxloss. Clarify both the reasons behind 20u and whether 10u give toxloss.

File: code/game/objects/items/storage/pill_packets.dm
Comment (novaepee) [inline code/game/objects/items/storage/pill_packets.dm:54]: Quick clot packet has peridaxon instead of quick clog.

File: code/game/objects/items/reagent_containers/pill.dm
Comment (novaepee) [reply code/game/objects/items/reagent_containers/pill.dm:249]: Alright, it was explained in Discord by MJP

> .25
> every tick
> you'll heal that off naturally unless you're premeedding on bica 24/7
> it's to stop people from doing 14/14/14 or whatever

File: code/game/objects/items/stacks/medical.dm
Comment (Losenis) [reply code/game/objects/items/stacks/medical.dm:179]: Wait is this seriously making kits even shittier than they already are?

Comment (Mrrpip) [reply code/game/objects/items/stacks/medical.dm:179]: We have that, it's called sutures.

File: code/modules/organs/limbs.dm
Comment (Pariah919) [reply code/modules/organs/limbs.dm:596]: healing with bandages/ointment sets wounds to treated, which heals you up a lot, it's a 75% prob so you don't heal rapidly from just slapping it on. I could make it a 100%

File: code/game/objects/machinery/vending/marine_vending.dm
Comment (Pariah919) [reply code/game/objects/machinery/vending/marine_vending.dm:792]: As I said on discord, I think these being pill bottles are fine as they're entirely utility medication and I think having more of them in a marines kit is fine as you trade a magazine or two for it.

File: code/game/objects/items/storage/pouch.dm
Comment (Pariah919) [reply code/game/objects/items/storage/pouch.dm:125]: I don't see how it needs to be only packets considering medkit pouches are the same storage.

File: code/game/objects/items/storage/pill_packets.dm
Comment (Pariah919) [reply code/game/objects/items/storage/pill_packets.dm]: fixed

File: code/game/objects/items/stacks/medical.dm
Comment (Pariah919) [reply code/game/objects/items/stacks/medical.dm]: changed to 3.5.

**Review by BraveMole — APPROVED** [review]
Ok good job.

So to resume what i said on disc:
This needs other maintainers approval
This needs another corp pr
You probably need to remove pill bottles laying on the map?
You probably need to lock chem not sure about this one, you choose

File: code/game/objects/items/storage/pill_packets.dm
Comment (TiviPlus) [inline code/game/objects/items/storage/pill_packets.dm:53]: dont capitalize the name in the descs

File: code/game/objects/items/storage/pouch.dm
Comment (TiviPlus) [inline code/game/objects/items/storage/pouch.dm:122]: No

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Also see discord comments

File: code/game/objects/items/storage/pouch.dm
Comment (Pariah919) [reply code/game/objects/items/storage/pouch.dm:122]: Medkit pouch is the same amount of slots, this doesn't really change anything in the long run.

**Comment by jroinc** [comment]
The medical nanites buff is not acceptable, balance-wise.

Lowering the blood cost by 4 means that medicines can keep up with it.

Which means that what you *think* is balanced allows ungas to make a buncha iron pills and convert it into meraderm at a 1/1 rate while never taking bloodloss.

**Pull request closed without merging**
